cmd/buildah/from.go

```diff
@@ -211,6 +212,16 @@ func fromCmd(c *cli.Context) error {
 	}
 	namespaceOptions.AddOrReplace(usernsOption...)
 
+	format := c.String("Format")
+	switch format {
+	case "oci":
+		format = imagebuildah.OCIv1ImageFormat
+	case "docker":
+		format = imagebuildah.Dockerv2ImageFormat
+	default:
```

Comment:
Same comment as from another review, can we merge these definition out of image.go and imagebuildah and into common.go?

Reply:
I want to get the other one merged first then I can clean this one up.